addons/website_slides/views/website_slides_templates.xml

```diff
@@ -182,6 +182,12 @@
                                 <span class="badge badge-pill" t-esc="header_object.nbr_infographics"/> Infographics
                             </a>
                         </li>
+                        <li t-if="header_object.nbr_webpages" class="nav-item">
+                            <a t-attf-href="/slides/#{slug(channel)}#{category and '/category/' + slug(category) or ''}/webpage"
+                                t-attf-class="nav-link#{slide_type == 'webpage' and ' active' or ''}">
```

Comment:
Maybe it's just because I'm not used to that notation it but I would prefer `' active' if slide_type == 'webpage' else ''`

(Same thing for the line above)

Reply:
Just copy/paste. Maybe tde will rewrite all this templates in its branch with the new homepage.